packages/api-server/rateLimit.js

```diff
@@ -0,0 +1,79 @@
+const { AccessGuard } = require("./lib/cache/guard");
```

Comment:
Can we move this file to src and write in TypeScript ?

And rename file name to `rate-limit.ts` ?

Reply:
updated